lib/glific/providers/maytapi/message.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Glific.Providers.Maytapi.Message do
+  @moduledoc """
+  Message API layer between application and Maytapi
+  """
+
```

Comment:
Shouldn't we add the behaviour `@behaviour Glific.Providers.MessageBehaviour`, we could empty callbacks for now too?